Automattic/wp-calypso
# FSE: move serialize_blocks fallback outside of the PHP namespace, and into its own file

Fixes regression where post_content was updated to an empty string on page update. First seen after adding a class namespace in https://github.com/Automattic/wp-calypso/pull/35074

The global function `serialize_blocks` is going to be available in WP 5.3 (not yet released), and we provide a fallback to take advantage of the server side processing.

This PR moves this to its own file, as we do intend for the fallback function to exist in the global namespace.

| Before  | After |
| ------------- | ------------- |
| ![before](https://user-images.githubusercontent.com/1270189/62404236-8dedfb00-b547-11e9-9a1d-ae1840543e77.gif) | ![after](https://user-images.githubusercontent.com/1270189/62404247-9b0aea00-b547-11e9-93e8-5ca47b30318e.gif)  |

### Testing Instructions
- Checkout this branch, and build files locally `npx lerna run build --scope='@automattic/full-site-editing'`
- Clone the Themes repository `git clone git@github.com:Automattic/themes.git`
- With a local Gutenberg environment, update the docker-compose file to map the following volumes:
```
    volumes:
//...
- ../wp-calypso/apps/full-site-editing/full-site-editing-plugin:/var/www/html/wp-content/plugins/full-site-editing-plugin
- ../themes/modern-business:/var/www/html/wp-content/themes/modern-business
```
- Start the Gutenberg env
- Activate the FSE plugin
- Activate Modern Business as the theme
- Create a new Page and select a starter template
- Make some edits and save
- Saves should be reflected in the editor and published post
- Navigation between the header/footer and page should still display all content
- No errors thrown from WordPress `docker-compose logs -f wordpress | grep php7`
